openff/evaluator/protocols/forcefield.py

```diff
@@ -561,7 +562,23 @@ def _execute(self, directory, available_resources):
             pdb_file.topology, unique_molecules=unique_molecules
         )
 
-        system = force_field.create_openmm_system(topology)
+        interchange = Interchange.from_smirnoff(
+            topology=topology, force_field=force_field
+        )
+
+        # Check for openmm incompatibilities
+        combine_nonbonded_forces = True
+        for collection in interchange.collections.values():
+            try:
+                collection.check_openmm_requirements(
+                    combine_nonbonded_forces=combine_nonbonded_forces
+                )
+            except AssertionError:
+                combine_nonbonded_forces = False
+
+        system = interchange.to_openmm(
+            combine_nonbonded_forces=combine_nonbonded_forces
+        )
```

Comment:
At some point we might want to think about how keywork arguments would be wired through here, including whether or not they should be. But that time is definitely not now

Reply:
Agreed.